src/index.html

```diff
@@ -1,12 +1,420 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo-image"
+        >
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="menu">
+          <li class="menu__item">
+            <a href="#" class="menu__link is-active">Apple</a>
+          </li>
```

Comment:
Looks like your menu is single block
```suggestion

      <nav class="menu">
        <ul class="menu__list">
          <li class="menu__item">
            <a href="#" class="menu__link is-active">Apple</a>
          </li>
```